testing/python/collect.py

```diff
@@ -705,6 +711,7 @@ def test_xfail(x):
         result = testdir.runpytest()
         result.stdout.fnmatch_lines('* 2 passed, 1 xpassed in *')
 
+    @pytest.mark.filterwarnings('ignore:Applying marks directly to parameters')
```

Comment:
those mark applications are all the same, can we assign a speaking name to that?